src/PaketRestoreTask/Program.cs

```diff
@@ -0,0 +1,58 @@
+using System;
+using System.IO;
+using System.Text;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.Build.Framework;
+using Microsoft.Build.Utilities;
+
+namespace Paket.Build.Tasks
+{
+    public class PaketRestoreTask : Task
```

Comment:
Just some crasy thinking: Would it make sense to think about extension points in paket?

Considering all those different use cases we might have soon (dotnetcore, toml, legacy, fable?) shouldn't we try to get things like these out of paket core into modules or something like that?